src/components/common/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { UseFormRegisterReturn } from 'react-hook-form';
+import { TextAreaWrapper, TextAreaLabel, TextAreaContainer } from './TextAreaStyle';
+
+interface ITextAreaProps {
```

Comment:
instead of listing all the possible attributes in props we should extend this interface from the default one